storage/src/factory.rs

```diff
@@ -245,6 +248,43 @@ impl BlobFactory {
         }
     }
 
+    pub fn new_backend_from_json(
```

Comment:
Can we reuse `new_backend` here? we can also reuse the handle of `#[cfg(feature = "backend-xxx")]`.

Reply:
Okay, let's only add the `feature` flag. :)